Solutions/Authomize/Analytic Rules/AWS_role_with_admin_privileges.yaml

```diff
@@ -0,0 +1,61 @@
+id: 734c00a0-a95b-44dd-9b69-d926ed44256d
+name: AWS role with admin privileges
+kind: Scheduled
+description: The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
```

Comment:
If queryPeriod is 30mins and queryFrequency is 5mins, same data is getting evaluated 6 times. This would lead to duplicate case creation or alert creation.
If there can be delay in ingesting events, we can have ingestion_time() based filter in the query to avoid this duplication.

Reply:
updated all queries with where ingestion_time() >= ago(30m)